daemon/api.go

```diff
@@ -910,6 +910,20 @@ func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated
 	return msg, updated, tasksets, nil
 }
 
+func verifySnapInstructions(inst *snapInstruction) error {
+	if inst.Action == "install" {
```

Comment:
Nitpic, switch/case on the inst.Action, this will also make the code less indented.